src/ucp/core/ucp_ep.inl

```diff
@@ -156,6 +156,20 @@ static UCS_F_ALWAYS_INLINE ucp_ep_ext_control_t* ucp_ep_ext_control(ucp_ep_h ep)
     return ucp_ep_ext_gen(ep)->control_ext;
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_ep_set_flags(ucp_ep_h ep, ucp_ep_flags_t flags)
+{
+    ucs_assert(ucs_async_is_blocked(&ep->worker->async));
+    ep->flags |= flags;
+}
+
+static UCS_F_ALWAYS_INLINE void
+ucp_ep_unset_flags(ucp_ep_h ep, ucp_ep_flags_t flags)
```

Comment:
minor: may be name it ```ucp_ep_clear_flags``` or ```ucp_ep_reset_flags```?
@yosefe what do you think?

Reply:
should be reset

Reply:
but, why do we need a function for this?
also maybe add single function
ucp_ep_update_flags(ep, flags_add, flags_remove)